dw-select-base-dialog.js

```diff
@@ -863,6 +863,10 @@ export class DwSelectBaseDialog extends DwCompositeDialog {
     if (_changedProperties.has("_newValueStatus")) {
       this._fire("new-value-status-changed", this._newValueStatus);
     }
+
+    if (_changedProperties.has('_activatedIndex')) {
```

Comment:
In general, I don't prefer to dispatch an event on property change; Instead, the event should be dispatched on user action. So, let's  review this flow together.